internal/expr/prepare.go

```diff
@@ -77,14 +80,63 @@ func prepareInput(ti typeNameToInfo, p *inputPart) (tm typeMember, err error) {
 			return nil, fmt.Errorf(`type %q not passed as a parameter (have "%s")`, p.sourceType.prefix, strings.Join(ts, `", "`))
 		}
 	}
-
+	if _, ok = info.(*sliceInfo); ok {
+		return nil, fmt.Errorf(`cannot use slice type %q outside of IN clause`, info.typ().Name())
+	}
+	if p.sourceType.name == "*" {
+		return nil, fmt.Errorf(`star type not allowed in standalone input`)
```

Comment:
Per earlier comment, what is a "standalone input"?

Reply:
Changed, see comment above.